packages/algolia-mesh/mesh/categories/getCategoryResults.ts

```diff
@@ -0,0 +1,34 @@
+import { MeshContext, QueryproductsArgs } from '@graphcommerce/graphql-mesh'
+import type { GraphQLResolveInfo } from 'graphql'
+import { getIndexName } from '../getIndexName'
+import { getCategoryResultsInput } from './getCategoryResultsInput'
+
+export async function getCategoryResults(
+  args: QueryproductsArgs,
```

Comment:
It should never bet Queryproductargs. I would expect something related to categories.

Reply:
```
filters: {name: { match: "men" }}
```